doc/mgr/influx.rst

```diff
@@ -0,0 +1,192 @@
+
+Example config file:
+
+::
+    [influx]
+        interval = 10
+        hostname = samplehost
+        username = admin
+        password = pass 
+        database = default 
+        port = 8086 
+        stats = osd, pool, cluster
+
+    [extended]
+        osd = op_latency, recovery_ops
+        cluster = op_latency
+
+--------
+Enabling 
+--------
+
+To enable the module the following steps must be taken in order:
+- Load the module by including this in the ceph.conf file 
+
+::
+
+    [mgr]
+        mgr_modules = influx 
+
+- Restart the mgr-daemon ``systemctl restart ceph-mgr@<mymonitor>`` 
+- Initialize the module to run every set interval  ``ceph mgr module enable influx``
+
+---------
+Disabling
+---------
+
+``ceph mgr module disable influx``
+
+---------
+Debugging 
+---------
+
+By default, a few debugging statments as well as error statements have been set to print in the log files. Users can add more if necessary.
+To make use of the debugging option in the module:
+
+- Add this to the ceph.conf file
+
+::
+
+    [mgr]
+        debug_mgr = 20  
+
+
+- Restart the ceph-mgr daemon ``systemctl restart ceph-mgr@<mymonitor>``
+- Use this command ``ceph tell mgr.<mymonitor> influx debug`` 
+- Check the log files. Users may find it easier to filter the log files using *mgr[influx]*
+
+-----
+Usage
+-----
+
+^^^^^^^^^^^^^^^^
+Default Counters
+^^^^^^^^^^^^^^^^
+
+**pool** 
+
++---------------+-----------------------------------------------------+
+|Counter        | Description                                         |
++===============+=====================================================+
+|bytes_used     | Bytes used in the pool not including copies         |
++---------------+-----------------------------------------------------+
+|max_avail      | The max available number of bytes in the pool       |
++---------------+-----------------------------------------------------+
+|objects        | Number of objects in pool                           |
++---------------+-----------------------------------------------------+
+|wr_bytes       | Number of bytes written in pool                     |
++---------------+-----------------------------------------------------+
+|dirty          | Number of bytes dirty in pool                       |
++---------------+-----------------------------------------------------+
+|rd_bytes       | Number of bytes read in pool                        |
++---------------+-----------------------------------------------------+
+|raw_bytes_used | Bytes used in pool including copies made            |
++---------------+-----------------------------------------------------+
+
+**osd**
+
++------------+------------------------------------+
+|Counter     | Description                        |
++============+====================================+
+|op_w        | Client write operations            |
++------------+------------------------------------+
+|op_in_bytes | Client operations total write size |
```

Comment:
"Client operations total write size" this does not conveys correct meaning .. 
Kindly rephrase